README.md

```diff
@@ -27,3 +27,13 @@ Generate various maps as png images:
 ```
 node tools/maps.js --input ./server/1.12.2/world --bounds -256 256 -256 256 --version 1.12.2
 ```
+
+Generate a village centered at 0, 0:
+```
+node tools/genvillage.js
```

Comment:
the input should be the build area bounds, and a path for the map
As part of the competition, village placement is also evaluated, its not necessarily at 0,0

Reply:
Gotcha. Will do.